src/main/java/lotto/model/Lotto.java

```diff
@@ -0,0 +1,36 @@
+package lotto.model;
+
+import lotto.constant.LottoNumConstant;
+
+import java.util.List;
+
+public class Lotto {
+    private final List<Integer> numbers;
+
+    public Lotto(List<Integer> numbers) {
+        validate(numbers);
+        validateDuple(numbers);
+        this.numbers = numbers;
+    }
+
+    private void validate(List<Integer> numbers) {
+        if (numbers.size() != LottoNumConstant.VALID_LOTTO_NUMBER_NUM.getValue()) {
+            throw new IllegalArgumentException();
+        }
+    }
+
+    private void validateDuple(List<Integer> numbers) {
```

Comment:
메소드 명은 축약하지말고 그대로 적는 것이 좋을 것 같아요!
```suggestion
    private void validateDuplicate(List<Integer> numbers) {
```